### DISCUSSION TITLE
More verbose logging

### DISCUSSION THREAD

**Discussion by BitTooth** [mikeoliphant/neural-amp-modeler-lv2#44] [Q&A]
Hi! I'm trying to use plugin with Ardour7 but it has some error. All I see in log is 
`LV2: Failed to instantiate plugin http://github.com/mikeoliphant/neural-amp-modeler-lv2`

I suspect that it could be the fact that I'm running it on the notebook without GPU. But is there a way to get more verbose logging to figure out what is going on?

**Comment by mikeoliphant** [comment]
No GPU is required.

I'm guessing that error means that the plugin failed to even get to the point where it would log anything.
